src/lib/actions.js

```diff
@@ -1,6 +1,7 @@
 import {store} from "./store";
 import {saveUserUrl} from "./fb";
 import {runLighthouse, fetchReports} from "./lighthouse-service";
+import "wicg-inert";
```

Comment:
I'm not against inert but an alternative approach is that focusing outside the nav bar implicitly closes it (it's what we do on Santa). Just my 2c and doesn't require the polyfill.

Reply:
I don't think that would prevent a screen reader from breaking outside of the side nav tho. Are you also marking the other areas as aria-hidden?